app/lists.rb

```diff
@@ -1,43 +1,33 @@
 module Listing
-  extend Discordrb::Commands::CommandContainer
-  
-  command :WingInitilise, description: "Initilise the array. In case of derps." do |event|
+	extend Discordrb::Commands::CommandContainer
+	
 	wing_list = Array.new
-  end
-
-  command :WingList, description: "Lists those who are requesting wing." do |event| #Simple, return the text file's contents. May consider making this a array. Also, admin lock this.
-    
-  end
-  
-  command :WingRemove, description: "Remove a person from the wing queue", min_args: 1, max_args: 1 do |event| #Remove someone who is on the wing list. Defo admin lock.
+	#The array that is wing_list is created.
 	
-  end
-
-  command :WingMe, description: "Need a wing? &WingMe" #Adds the person who used the command to the wing list.
-	to_wing = user.name # The user who typed &WingMe
-	wing_list.push(to_wing) #add them to the array-list
-	event.respond "Added to the Wing request queue."
-  end
+	command :winglist, description: "Lists those who are requesting wing." do |event|
+	#Simple, return the array's contents. Also, admin lock this.
+		
+	end
   
-  command :WingNext, description: "???" #prints top line of the text file, then removes it. Essentially shouting "Next"
-	tag_me = wing_list[0] #Save first value to tag_me
-	wing_list.uniq #Prints first value, then wipes from wing_list
-	event.respond wing_list.shift + " needs a wing"
-	event.respond "@" + tag_me + ", Wing request sent."
-  end
-end
+	command :wingremove, description: "Remove a person from the wing queue", min_args: 1, max_args: 1 do |event, target|
+	#Remove someone who is on the wing list. Defo admin lock.
+		
+	end
 
-#command :WingInitilise
-#wing_list - wing_list.new
-#
-#command :WingMe
-#to_wing = user.name
-#wing_list = wing_list push user.name
-#puts "Added to the Wing request queue."
-#
-#command :WingNext
-#tag_me = wing_list[0]
-#wing_list.uniq
-#puts wing_list.shift
-#puts " needs a wing"
-#puts "@" + tag_me + ", Wing request sent."
\ No newline at end of file
+	command :wingme, description: "Need a wing? &WingMe" do |event|
+	#Adds the person who used the command to wing_list.
+		wing_list.push(event.user.name)
```

Comment:
Test to see if the user is already in the array. You can do this easily in one line like this: 

```ruby
wing_list << event.user.name unless wing_list.any?(event.user.name)
```

If you wanted to notify the user that they are already on, wrap it in an `if` instead of the one-line `unless`.

(you can use `<<` to push a single item onto the array. In this case it's just a stylistic choice, but there is a small difference from `#push` in that `#push` allows you to push multiple things onto the array.)